nasa-gibs/worldview
# WV-3085 granule count slider sync issues

## Description

> Granule count slider goes out of sync with granule layer

## How To Test

1. `git checkout WV-3085-granule-count-slider-sync-issues`
2. open this [link](http://localhost:3000/?v=-145.93397634005998,-129.69262975142345,303.0015264567445,119.45618227527532&z=4&ics=true&ici=3&icd=10&l=VIIRS_SNPP_CorrectedReflectance_TrueColor_Granule(count=30)&lg=true&t=2024-03-14-T12%3A13%3A31Z)
3. dragging the granule count slider should behave normally.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
